### PR TITLE
HEEDLS-382 Fix error message for prompt name select

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
Changed the required error message for the prompt name. All changes in EditDetailsViewModel.cs are fixing mixed line endings in the file.

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs [hunk 1]
```diff
@@ -1,76 +1,76 @@
-namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
```
Comment (JBsoftwire) [inline DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs:1]: Did you make a change to this file? Or is this just a correction of line endings.

Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs:1]: Correction of mixed line endings

Comment (JBsoftwire) [reply DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsViewModel.cs:1]: All looks good to me then

**Review by stellake — APPROVED** [review]
Looks good - worth double checking that this fits with the error message guidelines / new error message structure 👍

**Pull request merged**